python/config_settings/config_settings.bzl

```diff
@@ -59,14 +62,39 @@ def construct_config_settings(name, python_versions):
         )
         matches_minor_version_names = [equals_minor_version_name]
 
+        default_micro_version = MINOR_MAPPING.get(minor_version)
+
         for micro_version in micro_versions:
             is_micro_version_name = "is_python_" + micro_version
+            if not default_micro_version:
```

Comment:
What case is this handling? AFAIK, the minor version is always in the MINOR_MAPPING dict. This if statement implies that is not true?

Is this supposed to be `if default_micro_version != micro_version` ?

Reply:
If the user is using this macro, which is not private, then we will still work. Do you think we don't need it?